src/mixins/allowable-actions-mixin.ts

```diff
@@ -14,8 +14,9 @@ export default class AllowableActionsMixin extends Vue {
   @Getter(useBusinessStore) isEntityFirm!: boolean
   @Getter(useBusinessStore) isEntitySoleProp!: boolean
   @Getter(useBusinessStore) isGoodStanding!: boolean
-  @Getter(useRootStore) isRoleStaff!: boolean
 
+  readonly IsAuthorized = IsAuthorized
+  readonly AuthorizedActions = AuthorizedActions
```

Comment:
I think you don't need these.

PS One impact of having this here is they will be available in all components that use this mixin, except you need to use `this.IsAuthorized()` instead of `IsAuthorized()`.

After removing these, you can remove "this." from lines 102 and 169 :)